cylc/flow/task_pool.py

```diff
@@ -836,9 +836,11 @@ def spawn_to_rh_limit(
         longer parentless, and/or hit the runahead limit.
 
         """
-        if not flow_nums or point is None:
-            # Force-triggered no-flow task.
-            # Or called with an invalid next_point.
+        if (
+            not flow_nums  # Force-triggered no-flow task
+            or point is None  # Reached end of sequence?
+            or point < self.config.start_point  # Warm start
```

Comment:
[not a blocker]
Do we want to continue to propogate the warm start terminology?